docs/src/gettingstarted/introduction/own-code/cli-install.md

```diff
@@ -6,52 +6,22 @@ aliases:
   - "/gettingstarted/own-code/cli-install.html"
 ---
 
-In the previous steps, you checked that the requirements on your computer were met and configured an SSH key on your Platform.sh account. Now all we have to do is install the CLI and you can access your projects from the command line.
+In the previous steps, you checked that the requirements on your computer were met. Now all we have to do is install the CLI and you can access your projects from the command line.
 
 {{< asciinema src="videos/asciinema/verify-cli-extended.cast" >}}
 
-1. **Install the CLI**
+## 1. Install the CLI
 
-    In your terminal, run the following command depending on your operating system:
+{{% cli-installation %}}
 
-{{< codetabs >}}
----
-title=Linux/macOS
-file=none
-highlight=bash
-markdownify=false
----
-
-curl -fsS https://platform.sh/cli/installer | php
-
-<--->
-
----
-title=Windows
-file=none
-highlight=bash
-markdownify=false
----
-
-curl.exe -f https://platform.sh/cli/installer -o cli-installer.php
-php cli-installer.php
-
-{{< /codetabs >}}
-
-
-1. **Authenticate and verify**
-
-   Once the installation has completed, run the CLI in your terminal with the command:
-
-   ```bash
-   platform
-   ```
+## 2. Test the CLI
 
-   Log in using a browser. Then take a moment to view some of the available commands with the command:
+To interact with your projects you need to be logged in.
+To open the authentication page, run:
```

Comment:
:hammer_and_wrench: 
```suggestion
To authenticate, run:
```

The page isn't something that's been referenced before, so it's less than optimal with a definite article. I would also not immediately find it clear that a "page" from a CLI refers to a website.